website_lazy_load_image/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,7 @@
+This module needs no manual configuration for new images.
+For existing images you need to change something on the page and save it for lazy loading to initialize.
+
+**WARNING**:
+
+When this module is uninstalled the images lazy loading is active will be invisible.
```

Comment:
The main issue IMO is that you are hacking and writing img on the fly in the editor. This leads to 2 problems:

1. the one you  mentioned
2. all the other images in the website out of the ones you edit are not taken in consideration

The [idea I had in mind](https://github.com/OCA/website/issues/425#issuecomment-365547007) was to transform the final HTML of the whole page so that we can handle this for ALL pages and ALL images in the site.

If you don't like this idea - perfectly fine :wink: - maybe you can make the editor hack conditional based on a config param or flag on the website?

This would leave the door open for an extension of top of it to deal w/ all images.

What do you think?

Reply:
Yes, transforming the HTML on transaction is better. But for now I can't spend time on it, this was for a customer and this was fine for them. When the transforming is made I think the client side transforming can be removed.